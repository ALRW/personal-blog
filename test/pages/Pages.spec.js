import { mount } from '@vue/test-utils'
import Index from '@/pages/index'

const IndexPage = mount(Index, {
  propsData: {
    data: {
      posts: [
        {
          title: 'Title',
          subtitle: 'Subtitle',
          id: 'id',
          path: '/',
          image: 'something.jpg'
        }
      ]
    }
  }
})

describe('Index Page', () => {
  it('is a vue component', () => {
    expect(IndexPage.isVueInstance()).toBeTruthy()
  })
  it('renders correctly', () => {
    expect(IndexPage.html()).toMatchSnapshot()
  })
})
