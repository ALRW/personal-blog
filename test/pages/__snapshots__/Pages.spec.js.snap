// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index Page renders correctly 1`] = `"<div class=\\"container\\" data=\\"[object Object]\\"><section class=\\"section\\"><div class=\\"columns is-multiline\\"></div></section></div>"`;
